### PR TITLE
ENH: Add somato data to testing and use "make source"

### PR DESCRIPTION
**Pull request by sappelhoff**

### CODE REVIEW

**Comment by jasmainak** [comment]
Cool ! Lots of issues with datalad, eh? :-)

**Comment by sappelhoff** [comment]
> Cool ! Lots of issues with datalad, eh? :-)

indeed - locally everything works! :-)  I am now trying to figure out a nice way to download the data ... but OSF is down :cry: 

Also: The way I uploaded `eeg_matchingpennies` as git-annex is fine for small datasets, but practically impossible for larger datasets ... or something that had derivatives like `freesurfer` --> simply because I was previously extracting file keys by hand! (this is no longer possible)

So now I'll try to fix datalad-osf

File: 01-import_and_filter.py [hunk 1]
```diff
@@ -106,8 +106,8 @@ def run_filter(subject, run=None, session=None):
         raw.resample(config.resample_sfreq, npad='auto')
 
     # Prepare the pipeline directory in /derivatives
-    fpath_out = op.join(config.bids_root, 'derivatives',
-                        'mne-study-template', subject_path)
+    deriv_path = op.join(config.bids_root, 'derivatives', 'mne-study-template')
```
Comment (agramfort) [inline 01-import_and_filter.py]: I am not too happy to see the name "study-template" in the code base. Imagine I fork the project, make my updated pipeline for my study. Now I need to potentially edit files unnecessarily. I could live if this path postfix is in the config though.

Comment (sappelhoff) [reply 01-import_and_filter.py]: okay, will fix

Comment (sappelhoff) [reply 01-import_and_filter.py]: see https://github.com/mne-tools/mne-study-template/pull/48/commits/7a1168ca09201c4454c32c1600ea3be4ce91e8f3

**Comment by sappelhoff** [comment]
The datalad issues are solved!

- MNE-somato-data git-annex: https://github.com/sappelhoff/MNE-somato-data
- Docs on how to do it: [my GSoC blog](https://blogs.python-gsoc.org/en/sappelhoffs-blog/eleventh-week-of-gsoc-some-more-datalad-complete-and-automatic-flow-now/)

once the CIs are green, I think this is ready to be merged

**Comment by jasmainak** [comment]
okay! CIs are not yet green though

**Comment by sappelhoff** [comment]
ping @jasmainak  it's green :-)

**Comment by agramfort** [comment]
Beautiful

**Comment by jasmainak** [comment]
excellent! great progress

**Pull request merged**